cfgov/jinja2/v1/_includes/organisms/filter-filterable-list.html

```diff
@@ -0,0 +1,159 @@
+{# ==========================================================================
+
+   filter_filterable_list.render()
+
+   ==========================================================================
+
+   Description:
+
+   Create an expandable of filters when given:
+
+   settings:            Array of objects describing a form structure.
+                        Example nested form:
+                        {
+                            'type': 'form',
```

Comment:
That's a good point. Made some adjustments in https://github.com/cfpb/cfgov-refresh/commit/9e3f2b6f9de8b2490397fc2f5c1d752caaccfa62